src/timers.cc

```diff
@@ -188,22 +188,9 @@ std::string DateTimeString(bool local) {
   std::size_t written;
 
   if (local) {
```

Comment:
given we only ever set `local` to true, i think you can simplify this and assume that we want the local time always.